packages/remix-dev/__tests__/cli-test.ts

```diff
@@ -134,6 +135,11 @@ describe("remix cli", () => {
   });
 
   describe("the create command", () => {
+    beforeAll(async () => {
+      await fsp.rm(TEMP_DIR, { force: true, recursive: true });
+      await fsp.mkdir(TEMP_DIR, { recursive: true });
```

Comment:
```suggestion
      await fse.emptyDir(TEMP_DIR);
```

For stuff like this I love using [fs-extra](https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/emptyDir.md). We should probably just default to using that everywhere instead of fs and fs/promises.